logfire/_internal/integrations/httpx.py

```diff
@@ -152,24 +151,26 @@ def content_type_header_string(self) -> str:
     def content_type_is_json(self):
         return is_json_type(self.content_type_header_string)
 
-    @property
-    def content_type_is_text(self):
```

Comment:
I wrote this `is_text_type` stuff based on arbitrary decisions I made myself, happy to get rid of all things related. Users should check the content type themselves if they want to. We will only check if bytes can be decoded as text.